### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by steven-k1** [swellstores/community#2] [Announcements]
Hello everyone, 

We want to welcome you to our community discussions, the home for all things Swell!

This is a place to ask questions, discuss issues, share what you’re working on, and discuss ideas for how Swell can be improved.

- 🤔 Post a question in [Help](https://github.com/orgs/swellstores/discussions/categories/help) if you can't find the answers you need in our [Documentation](https://developers.swell.is/) or [Help Center](https://www.swell.is/help).
- 🐞 If you encounter issues or bugs with the platform or one of our open-source tools, post in the [Help](https://github.com/orgs/swellstores/discussions/categories/help) channel and add the `bug` label.
- 📣 The Swell team will post updates and requests for feedback in this announcements channel.
- 👀 If you created something you think is interesting and want to share it, post in the [Show and tell](https://github.com/swellstores/community/discussions/categories/show-and-tell) channel. It could be a blog, a tool, a storefront, or something that defies categorization! We'll be doing a regular round-up of our favorites, and adding them to the [Awesome Swell](https://github.com/swellstores/community) link list.

As a reminder, the [code of conduct](https://github.com/swellstores/community/blob/8ec641af892875ae40e06bcf615263b0b14e1ae0/CODE_OF_CONDUCT.md) applies to all discussions. This is a space where everyone is welcome and should feel comfortable.

See you around! 👋

**Comment by joshuavoydik** [comment]
This is great.

**Comment by jesseswell** [comment]
Awesome!

**Comment by ericingram** [comment]
Heck yeah

**Comment by siphosenkosindhlovu** [comment]
Hope you actually answer questions this time.

**Comment by logeshswell** [comment]
Awesome!

**Comment by ana-estrela** [comment]
Love this!

**Comment by DabstiX** [comment]
So far this has been stellar, long way to go and just getting started.
